.github/workflows/ci.yml

```
@@ -17,37 +17,37 @@ jobs:
-            ./build/*.tar.gz
-            ./build/*.nupkg
-            ./build/*.snupkg
-
-      - name: Push NuGet package
-        if: github.ref_type == 'tag'
-        run: |
-          dotnet nuget push ./build/*.nupkg --source https://api.nuget.org/v3/index.json --api-key ${{ secrets.NUGET_IMPOSTOR }}
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)